cluster/addons/ip-masq-agent/ip-masq-agent.yaml

```diff
@@ -21,6 +21,13 @@ spec:
       containers:
       - name: ip-masq-agent
         image: gcr.io/google-containers/ip-masq-agent-amd64:v2.0.1
+        resources:
+          limits:
+            cpu: 10m
+            memory: 8Mi
```

Comment:
For memory, we're best off to set request and limit to the same.  The repercussions are more severe for getting it wrong on a busy machine.